### PR TITLE
Fix check cli

### PR DESCRIPTION
**Pull request by chinhuang007**
The graph check cli is broken with the addition of
experimental ops returned from get_frontend_coverage.
Also fixed the logging not working issue because of
earlier logging level set to warning.

### CODE REVIEW

**Comment by fumihwh** [comment]
Maybe returning 3 results is not a good feature..

**Comment by chinhuang007** [comment]
It is okay to return three results so gen_opset.py could put "experimental" tags in opset_version.py. We should think about making our helper methods, considered as internal APIs, more specific and stable. We probably should not freely add or remove inputs and outputs otherwise it might cause unforeseen errors, especially there is no "complete test" to verify every piece of onnx-tf code.

**Comment by fumihwh** [comment]
Agree. Add or remove inputs and outputs is risky. We should not do this. In this case, dict would be better than three result.

**Comment by chinhuang007** [comment]
@fumihwh Thanks for the comments. I believe you suggested to change the helper method to return a dict so we could add more content later without affecting existing code. Please review.

File: onnx_tf/opr_checker.py [hunk 1]
```diff
@@ -13,6 +14,7 @@
 from onnx_tf.pb_wrapper import TensorflowNode
 
 warnings.filterwarnings('ignore')
+reload(logging)
```
Comment (fumihwh) [inline onnx_tf/opr_checker.py]: BTW, what reload logging for?

Comment (chinhuang007) [reply onnx_tf/opr_checker.py]: The reload clears possibly configuration by other modules and makes sure the logging setting in our code is actually used, in this case level=logging.DEBUG. In my test, the logging level in use is always the default WARNING if there is no reload.

**Pull request merged**